TestVectorsAwsCryptographicMaterialProviders/runtimes/java/build.gradle.kts

```diff
@@ -20,7 +20,7 @@ var props = Properties().apply {
 var dafnyVersion = props.getProperty("dafnyVersion")
 
 group = "software.amazon.cryptography"
-version = "1.6.0-SNAPSHOT"
+version = "1.7.0"
```

Comment:
doesn't really matter but I think for testvectors this always remains a snapshot

Reply:
I'll fix this before finishing the MCM

Reply:
Thanks for pointing it out.